packages/shared/components/MenuLogin/types.ts

```diff
@@ -19,7 +19,7 @@ export type LoginItem = {
   login: string;
 };
 
-export type MenuSshLoginProps = {
+export type MenuLoginProps = {
   onOpen: () => LoginItem[];
```

Comment:
Would you like to rename this prop to sth like `getLoginItems`? IMO `onOpen` is really confusing

Reply:
Ah, good call, I was confused by this as well but I completely forgot to change it.